src/components/Image/Image.mdx

```diff
@@ -27,71 +27,81 @@ import { changelogData } from "./imageChangelogData";
 
 ## Table of contents
 
-- {<Link href="#overview" target="_self">Overview</Link>}
-- {<Link href="#component-props" target="_self">Component Props</Link>}
-- {<Link href="#accessibility" target="_self">Accessibility</Link>}
-- {<Link href="#credit-and-caption" target="_self">Credit and Caption</Link>}
-- {<Link href="#sizes" target="_self">Sizes</Link>}
-- {<Link href="#aspect-ratios" target="_self">Aspect Ratios</Link>}
-- {<Link href="#types" target="_self">Types</Link>}
-- {<Link href="#html-attributes" target="_self">HTML Attributes</Link>}
-- {<Link href="#fallback-image" target="_self">Fallback Image</Link>}
-- {<Link href="#lazy-loading" target="_self">Lazy Loading</Link>}
-- {<Link href="#changelog" target="_self">Changelog</Link>}
+- [Overview](#overview)
+- [Component props](#component-props)
+- [Accessibility](#accessibility)
+- [Credit and caption](#credit-and-caption)
+- [Sizes](#sizes)
+- [Aspect ratios](#aspect-ratios)
+- [Types](#types)
+- [HTML attributes](#html-attributes)
+- [Fallback image](#fallback-image)
+- [Lazy loading](#lazy-loading)
+- [Changelog](#changelog)
 
 ## Overview
 
-<Description of={ImageStories.WithControls} />
+<Banner
+  content={
+    <>
+      <strong>NOTE:</strong> This page mostly describes the Reservoir{" "}
+      <code>Image</code> component. If you'd like to use Next.js's{" "}
+      <code>Image</code>, see the recommendations for styling it{" "}
+      <a href="../?path=/docs/development-guide-using-next-js-image--docs">
```

Comment:
I've been updating the relative urls to have one dot. Two dots work but not always locally. Have you noticed this?